modules/preview-react/loading-sparkles/lib/LoadingSparkles.tsx

```diff
@@ -49,42 +30,37 @@ const LOADING_ANIMATION = keyframes({
   },
 });
 
-const loadingSparklesIconStyles = createStyles({
-  animationDuration: `${ANIMATION_DURATION_MS}ms`,
-  animationFillMode: 'both',
-  animationIterationCount: 'infinite',
-  animationName: LOADING_ANIMATION,
-  animationTimingFunction: 'ease-in-out',
-  '.wd-sparkle-fill': {
-    fill: AI_COLORS.dragonFruit400,
-  },
-  '&:nth-child(1)': {
-    animationDelay: '0ms',
-  },
-  '&:nth-child(2)': {
-    animationDelay: `calc(${ANIMATION_DURATION_MS}ms * (1/3))`,
-  },
-  '&:nth-child(3)': {
-    animationDelay: `calc(${ANIMATION_DURATION_MS}ms * (2/3))`,
-  },
-});
-
 /**
  * An individual loading sparkle. ✨
  */
 const Sparkle = () => {
-  return (
-    <SystemIcon
-      icon={sparkleIcon}
-      cs={[loadingSparklesIconStyles]}
-      size={cssVar(system.space.x3)}
-    />
-  );
+  return <SystemIcon icon={sparkleIcon} size={cssVar(system.space.x3)} />;
 };
 
-const loadingSparklesStyles = createStyles({
-  display: 'inline-flex',
-  gap: px2rem(1),
+const loadingSparklesStencil = createStencil({
+  base: {
+    display: 'inline-flex',
+    gap: px2rem(1),
+    '> span': {
```

Comment:
We're starting to use `data-part` and this will be easier to refactor later if necessary. 

If we needed to swap this from a `span` to a `div`, this would be agnostic and easier for the future. 

```suggestion
    '& [data-part="sparkle"]':{
```